### PR TITLE
Elevator hell

### PR DESCRIPTION
**Pull request by CrabbytheCrab**
## About The Pull Request

FUCK YOU KIRIE! IF YOU'RE DUMB ENOUGH TO PLAY A MAP THIS WEEKEND, YOU'RE A DUMB ENOUGH SHMUTZ TO LUCAS’ ELEVATOR HELL! BAD DESIGN, ELEVATORS THAT BREAK DOWN, BEES! IF YOU THINK YOUR GOING TO FIND A GOOD ROUND AT ELEVATOR HELL, YOU CAN KISS MY ASS! IT'S MY BELIEF THAT YOU'RE SUCH A STUPID MOTHERFUCKER THAT YOU'LL FALL FOR THIS BULLSHIT. GUARANTEED! IF YOU FIND A BETTER MAP, YOU CAN SHOVE IT UP YOUR UGLY ASS. YOU HEARD ME RIGHT, SHOVE IT UP YOUR UGLY ASS! BRING YOUR AGENTS, BRING YOUR CLERKS, BRING YOUR AI ASSISTANT, I'LL FUCK HER! THAT'S RIGHT I'LL FUCK YOUR AI ASSISTANT! BECAUSE AT ELEVATOR HELL, YOU'RE FUCKED SIX WAYS TO SUNDAY! TAKE A HIKE TO ELEVATOR HELL! HOME OF CHALLENGE [CENSORED], THAT'S RIGHT, CHALLENGE [CENSORED]. HOW DOES IT WORK? IF YOU CAN [CENSORED] 6 FEET IN THE AIR STRAIGHT AND NOT GET [CENSORED], YOU GET NO WAIT TIMES. DON'T WAIT, DON'T DELAY, DON'T FUCK WITH ME OR I'LL RIP YOUR NUTS OFF! ONLY AT ELEVATOR HELL, THE ONLY MAP THAT TELLS YOU TO FUCK OFF. HURRY UP, ASSHOLE! THIS EVENT ENDS THE MINUTE YOU WRITE ME A REVIEW AND IT BETTER NOT BOUNCE OR YOU'RE A DEAD MOTHERFUCKER. GO TO HELL. ELEVATOR HELL, LC13’S FILTHIEST AND EXCLUSIVE MAP OF THE MEANEST SON OF BITCHES IN THE GAME SS13, GUARANTEED!!

Elevator hell is a map that uses elevators to have 3 wacky floors. Also has tubes for non elevator transport to each floor but its slower.

## Why It's Good For The Game

to make people that have ptsd of e corp shit themselves but also be a good map from the pop needed for b corp to d corp that uses elevators. also removes a window that was in a wall from a corp
## Changelog
:cl:
add: Added a new wacky map
imageadd: replaced the gray and black mixed tiles with sprites reflecting the whole versions
code: added elevator 2
/:cl:

### CODE REVIEW

**Comment by Gboster-0** [comment]
decent copy-pasta, map looks fine

File: _maps/data/logs/2023/10/24/round-11.55.28/perf-NULL-Meta Station.csv
Comment (Kitsunemitsu) [inline _maps/data/logs/2023/10/24/round-11.55.28/perf-NULL-Meta Station.csv:1]: Why is this here?

File: _maps/elevator_hell.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+	"version": 1,
+	"map_name": "Facility E2-098 Elevator Hell",
```
Comment (Kitsunemitsu) [inline _maps/elevator_hell.json:3]: Don't call it E2.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
I'll go over mapping issues on a thread on the discord.

**Comment by CrabbytheCrab** [comment]
waiting for a new review of this

**Comment by Retrino** [comment]
reviewed in the thread

**Comment by CrabbytheCrab** [comment]
how to compact

**Comment by CrabbytheCrab** [comment]
should I just close this at somepoint?

**Comment by Kitsunemitsu** [comment]
I thought about it, and I'm not super interested in a new map at this moment. 
Apologies. 

You can talk with skits about it on the discord, and if they approve I'll reopen.

**Pull request closed without merging**